clang/lib/CodeGen/Targets/NVPTX.cpp

```diff
@@ -9,6 +9,7 @@
 #include "ABIInfoImpl.h"
 #include "TargetInfo.h"
 #include "llvm/IR/IntrinsicsNVPTX.h"
+#include <cstdint>
```

Comment:
Is this necessary?

Reply:
probably not, and if it is I believe there's an llvm wrapper header you're supposed to use instead